core/client/eth/client.go

```diff
@@ -93,19 +97,11 @@ func (c *Client) UpdateEthereumConfig(ethConfig *types.EthereumConfig) error {
 		return fmt.Errorf("updated chain ID does not matchthe one set during start up, expected %v got %v", ethConfig.ChainID(), chainID)
 	}
 
-	// if err := c.verifyStakingContract(context.Background(), ethConfig); err != nil {
-	// 	return fmt.Errorf("failed to verify staking bridge contract: %w", err)
-	// }
-
-	// if err := c.verifyVestingContract(context.Background(), ethConfig); err != nil {
-	// 	return fmt.Errorf("failed to verify vesting bridge contract: %w", err)
-	// }
-
-	if err := c.verifyCollateralContract(context.Background(), ethConfig); err != nil {
+	if err := c.verifyCollateralContract(ctx, ethConfig); err != nil {
```

Comment:
We will likely want to disable that for the new bridge seeing that the code will change and bytecode will be different.

Reply:
Sure, not doing it right now, to avoid scope creep for the PR. But, I'll take good notes of that.

Reply:
I saved it here:
https://github.com/vegaprotocol/vega/issues/10623